detector/src/main/java/com/synopsys/integration/detector/base/DetectorStatusCode.java

```diff
@@ -8,6 +8,7 @@ public enum DetectorStatusCode {
     EXCLUDED("Detector type was excluded."),
     EXECUTABLE_FAILED("During extraction, one or more executables did not execute successfully."),
     EXTRACTION_FAILED("During extraction, one or more exceptions were encountered."),
+    EXECUTABLE_TERMINATED_LIKELY_OUT_OF_MEMORY("An executable was terminated likely by the runtime environment due to low memory."),
```

Comment:
Note to reviewers:

Previously `OUT_OF_MEMORY_EXCEPTION("The detector process has encountered an out memory exception under the runtime environment.")` was discussed as the status name/description.

I went with this alternative because there is a chance that something else could be the reason for process termination. For example, if you issue `kill -s KILL <pid>` on either Mac or Linux the exit code for the killed process will also be `137`.

Please feel free to chime in on this.

Reply:
Do we mean "An executable was terminated by the runtime environment _likely_ due to low memory."?

Reply:
What I was trying to highlight with that sentence is that it wasn't necessarily the runtime environment (OS) which terminated the subprocess. I do realize it sounds a bit odd.

Reply:
That is a tricky message, when the termination cause is not certain.
Perhaps a slight rewording:
"An executable was terminated, potentially due to a runtime environment low memory condition."